packages/contentstack-bootstrap/src/bootstrap/interactive.ts

```diff
@@ -99,3 +99,14 @@ export async function inquireLivePreviewSupport() {
   });
   return livePreviewEnabled;
 }
+
+export async function continueBootstrapCommand() {
+  const {shouldContinue} = await inquirer.prompt({
```

Comment:
[MINOR] Alignments/prettier format missing it seems.

Reply:
added